packages/manager/apps/web/client/app/email-domain/email/redirection/email-domain-email-redirection.controller.js

```diff
@@ -45,13 +43,25 @@ angular.module('App').controller(
         ],
       ];
 
-      return this.$q
-        .all(
-          map(this.redirections, ({ id }) =>
-            this.WucEmails.getRedirection(this.$stateParams.productId, id),
-          ),
-        )
-        .then((data) => dataToExport.concat(map(data, (d) => [d.from, d.to])))
+      return this.iceberg('/email/domain/:domain/redirection')
+        .query()
+        .expand('CachedObjectList-Pages')
+        .execute({
+          domain: this.$stateParams.productId,
+        })
+        .$promise.then((result) => {
+          result.data
+            .sort((a, b) => {
+              if (a.from === b.from) {
```

Comment:
i think the comparator could be simplified :
```sort((a, b) => a.from - b.from || a.to - b.to)```

Reply:
`from` and `to` are email addresses. I tried to substract strings, but it doesn't seem to work

Reply:
ok then it would be better to use String.localeCompare (https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/String/localeCompare) because :
```
'a' < 'b' == true
'a'.localeCompare('b') == -1
'f' < 'é' == true // expected false
'f'.localeCompare('é') == 1
```

Reply:
re-reading your comment i guess it should be fine if it's only email address since allowed characters are restricted

Reply:
I thought of another point: case sensitivity. I don't know if the manager/database allows case sensitive input, but `"foo" === "FOO"` returns `false`

Reply:
well, did test, input is lowercased, so it should be OK